### PR TITLE
Update video.md

### PR DESCRIPTION
**Pull request by JamesH65**
Updated the CEA table with the rest of the supported modes.

### CODE REVIEW

**Comment by andrum99** [comment]
Might be worth adding these new modes as seperate table, noting that they are only possible on the Pi 4B.

File: configuration/config-txt/video.md [hunk 1]
```diff
@@ -162,6 +162,8 @@ Together with `hdmi_group`, `hdmi_mode` defines the HDMI output format.
 
 To set a custom display mode not listed here, see [this thread](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=24679).
 
+Note that not all modes are available on all models. 
+
```
Comment (6by9) [inline configuration/config-txt/video.md:166]: Ought to reference that these VIC codes come from CTA-861-G

File: configuration/config-txt/video.md [hunk 2]
```diff
@@ -225,6 +227,54 @@ These values are valid if `hdmi_group=1` (CEA):
...
+| 88 | 2560x720 | 30Hz | 64:27 aspect ratio |
+| 89 | 2560x720 | 50Hz | 64:27 aspect ratio |
+| 90 | 2560x720 | 60Hz | 64:27 aspect ratio |
+| 91 | 2560x720| 100Hz | 64:27 aspect ratio |
+| 92 | 2560x720 | 120Hz | 64:27 aspect ratio |
+| 93 | 2160p | 24Hz |  |
+| 94 | 2160p | 25Hz |  |
+| 95 | 2160p | 30Hz |  |
+| 96 | 2160p | 50Hz |  |
+| 97 | 2160p | 60Hz |  |
```
Comment (6by9) [inline configuration/config-txt/video.md:267]: Referencing the spec, these are 16:9 aspect ratio

File: configuration/config-txt/video.md [hunk 3]
```diff
@@ -225,6 +227,54 @@ These values are valid if `hdmi_group=1` (CEA):
...
+| 98 | 4096x2160 | 24Hz | 256:135 aspect ratio |
+| 99 | 4096x2160 | 25Hz | 256:135 aspect ratio |
+| 100 | 4096x2160 | 30Hz | 256:135 aspect ratio |
+| 101 | 4096x2160 | 50Hz | 256:135 aspect ratio |
+| 102 | 4096x2160 | 60Hz | 256:135 aspect ratio |
+| 103 | 2160p | 24Hz | 64:27 aspect ratio |
+| 104 | 2160p | 25Hz | 64:27 aspect ratio |
+| 105 | 2160p | 30Hz | 64:27 aspect ratio |
+| 106 | 2160p | 50Hz | 64:27 aspect ratio |
+| 107 | 2160p | 60Hz | 64:27 aspect ratio |
```
Comment (6by9) [inline configuration/config-txt/video.md:277]: If we're redoing the table,  you may as well create a column for the aspect ratio, and move things like rb and doubling/quadrupling into a notes field. At the moment the notes field is a mismash.
As per spec, you also have pixel aspect ratio on each of these modes too - they aren't all square pixels.

Comment (JamesH65) [reply configuration/config-txt/video.md]: Not made changes related to pixel aspect ratio, not sure they are necessary?

File: configuration/config-txt/video.md [hunk 4]
```diff
@@ -158,168 +158,218 @@ The `hdmi_group` command defines the HDMI output group to be either CEA (Consume
...
+| 79 | 2560x1600 | 85Hz | 16:10 | |
+| 80 | 2560x1600 | 120Hz | 16:10 | reduced blanking |
+| 81 | 1366x768 | 60Hz | 16:9 |  |
+| 82 | 1920x1080 | 60Hz | 16:9 | 1080p |
+| 83 | 1600x900 | 60Hz | 16:9 | reduced blanking |
+| 84 | 2048x1152 | 60Hz | 16:9 | reduced blanking |
+| 85 | 1280x720 | 60Hz | 16:9 | 720p |
+| 86 | 1366x768 | 60Hz | 16:9 | reduced blanking |
+
+Note that there is a [pixel clock limit](https://www.raspberrypi.org/forums/viewtopic.php?f=26&t=20155&p=195443#p195443).The highest supported mode on models prior to the Raspberry Pi 4 is 1920x1200 at 60Hz with reduced blanking, whilst the Raspberry Pi 4 can support up to 4096x2160 at 60Hz.
```
Comment (lurch) [inline configuration/config-txt/video.md:372]: > whilst the Raspberry Pi 4 can support up to 4096x2160 at 60Hz.

IIRC that's if only a single display-output is being used? Is it worth mentioning the maximum resolution/frequency if both HDMI outputs are being used?

**Comment by lurch** [comment]
I've just noticed that near the top of this page it says:
"Further details on the syntax and alternatives mechanisms can be found on the conditionals page in the HDMI section of the documentation."
Should this be reworded slightly to:
"Further details on the syntax and alternatives mechanisms can be found in the HDMI section on the conditionals page of the documentation."
?

File: configuration/config-txt/video.md (hunk 4 above)
Comment (JamesH65) [reply configuration/config-txt/video.md:372]: Good idea. Will do.

**Comment by JamesH65** [comment]
Fixed up Andrew's comments.

File: configuration/config-txt/video.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ On older Pi models, the composite behaviour remains the same.
 
 ## HDMI mode options
 
-**Note for Raspberry Pi4B users:** Because the Raspberry Pi 4B has two HDMI ports, some HDMI commands can be applied to either port. You can use the syntax `<command>:<port>`, where port is 0 or 1, to specify which port the setting should apply to. If no port is specified, the default is 0. If you specify a port number on a command that does not require a port number, the port is ignored. Further details on the syntax and alternatives mechanisms can be found on the [conditionals page](./conditional.md) in the HDMI section of the documentation. 
+**Note for Raspberry Pi4B users:** Because the Raspberry Pi 4B has two HDMI ports, some HDMI commands can be applied to either port. You can use the syntax `<command>:<port>`, where port is 0 or 1, to specify which port the setting should apply to. If no port is specified, the default is 0. If you specify a port number on a command that does not require a port number, the port is ignored. Further details on the syntax and alternatives mechanisms can be found in the HDMI section on the [conditionals page](./conditional.md) page of the documentation.ion.
```
Comment (lurch) [inline configuration/config-txt/video.md:42]: You've now got the word "page" in there twice :wink: 
`section on the [conditionals page](./conditional.md) page of the`

Comment (JamesH65) [reply configuration/config-txt/video.md:42]: Bugger, I read that multiple times as well!

**Pull request merged**